src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/StringCredentials.java

```diff
@@ -9,7 +9,7 @@
 @Describable("Secret text")
 public class StringCredentials extends BaseStandardCredentials {
 
-    public Control secret = control("secret");
+    public Control secret = control(by.name("_.secret"));
```

Comment:
Ok, so if `name` attribute was unchanged, why are you changing it here? ` "_.secret"` != `"secret"`.

Anyway, my question is: is this working on 1.x? If so, I'm ok.